test/index.test.js

```diff
@@ -23,30 +24,39 @@ describe('markdownify', () => {
     nock('https://api.github.com')
       .post('/app/installations/321696/access_tokens')
       .reply(200, {token: 'test'})
-
-    nock('https://api.github.com')
```

Comment:
let me know if you want me to remove the nock chaining here. I just wanted to make sure you know about it :)